docs/man_pages/project/configuration/resource-create.md

```diff
@@ -0,0 +1,43 @@
+resource create
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder resource create [--icon <File Path>] [--splash <File Path>] [--force]`
+
+Creates image resources for all mobile platforms from a single high-resolution image and saves them to the `App_Resources` folder of the current project. The image source must be a PNG image.
+
+<% if(isConsole && isMobileWebsite) { %>
+WARNING: This command is not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help resource create`
+<% } %>
+<% if((isConsole && (isCordova || isNativeScript)) || isHtml) { %>
```

Comment:
My comment for this line got outdated, so here it is again:

Insert an isHtml block with the image prerequisites:

```
### Prerequisites

* Verify that the image source for your icons is an at least XxY pixels `PNG` image.
* Verify that the image source for your splash screens is an at least XxY pixels `PNG` image. Any logos should be centered.
```